### ISSUE TITLE
Ship shuts down when I join Urbit Community chats

### ISSUE THREAD

**Issue by Sidfyl** [urbit/urbit#4062]
I've been having some trouble with the UC group.  I can join the group itself, but if I try to subscribe to any of its chats, my ship goes down with a bail:meme.

I've alredy tried to |pack and used the experimental binary with its recovery commands.

I'm on hash rd3oe, latest binary.

**Comment by matildepark** [comment]
`|unlink %chat-cli` and try again?

**Comment by Sidfyl** [comment]
Nope, ames'd to death

```
bail: meme
bailing out
pier: serf error: end of file
~rivpyl-sidfyl:dojo> 
address 0x458 out of loom!
loom: [0x200000000 : 0x280000000)
Assertion '0' failed in noun/events.c:129

bail: oops
bailing out
Assertion failed: (0), function u3m_bail, file noun/manage.c, line 677.
Abort trap: 6
```

**Comment by Sidfyl** [comment]
don't know if this is relevant, but I get a lot of this preceding my crashes:

```
%chat-hook-resubscribe
[%mailbox-kick i=%~.mailbox t=[i=~.~dasfeb t=/smol-chat-1423]]
%chat-hook-resubscribe
[%mailbox-kick i=%~.mailbox t=[i=~.~timluc-miptev t=/technical-1266]]
%chat-hook-resubscribe
[%mailbox-kick i=%~.mailbox t=[i=~.~timluc-miptev t=/culture---theory-7667]]
%chat-hook-resubscribe
[%e %clogged '1607128078858-592873' 2]
[%e %clogged '1607136904702-aabcec' 2]
[%mailbox-kick i=%~.mailbox t=[i=~.~timluc-miptev t=/social-hangout-8926]]
%chat-hook-resubscribe
[%mailbox-kick i=%~.mailbox t=[i=~.~timluc-miptev t=/macro---money-3988]]
%chat-hook-resubscribe
[%e %clogged '1607128078858-592873' 9]
[%e %clogged '1607136904702-aabcec' 9]
```

**Comment by Sidfyl** [comment]
I ended up breaching after some flagday trouble, so I can no longer troubleshoot this. should I close?

**Comment by Fang-** [comment]
Guessing this might have been a case solved by #4105.

**Comment by joemfb** [comment]
Thanks, @Sidfyl. Closing in favor of #4105 (which was likely the fix) and/or #4182, which tracks bail:meme in general.
